### DISCUSSION TITLE
Courseplay control point

### DISCUSSION THREAD

**Discussion by CZEKenyCZE** [Courseplay/courseplay#7729] [Q&A]
According to which NODE in the model is Courseplay controlled??

AUTODRIVE is fine in the middle... but Couseplay is on the left
Why??

![Farming Simulator 19 08 12 2022 18_30_09](https://user-images.githubusercontent.com/81824516/206552312-90cc85a4-5d81-4add-b0fe-9905e0d5dc24.png)

**Comment by Tensuko** [comment]
Looks like an offset.

**Reply by CZEKenyCZE** [reply]
What offset??

**Reply by Tensuko** [reply]
On the HUD.
But you may look here:
https://github.com/Courseplay/courseplay/issues/4595#issuecomment-739723986

**Comment by CZEKenyCZE** [comment]
Ouuu,   yes man.... thank you!!!!!!!!!!!!
